docs/team/irfan.md

```diff
@@ -3,44 +3,74 @@ layout: page
 title: Irfan's Project Portfolio Page
 ---
 
-### Project: AddressBook Level 3
+### Project: CallMeMaybe
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+CallMeMaybe (CMM) is a **desktop app** centered for Telemarketers in aiding them in customer contact management.
+The in-built tracking functionality serves as a reminder to reach back on previously unreachable customers.
+Importing and exporting of existing customer database is also supported by CMM to facilitate team-based environments.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. This It is written in Java, and has about
+16 kLoC. The development of this software was used as a medium for students to apply Software Engineering principles
+taught during the Software Engineering Module CS2103T.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added Importing functionality
+  * What it does: Allows users to input contacts CMM's database quickly. CMM currently imports from Excel Files only
+  * Justification: Based on the user story, a telemarketer receives daily call list in an Excel sheet. It required an 
```

Comment:
In the sentence, 'It required an...', what does it refer to? The sentence seems a bit unclear.